### ISSUE TITLE
AI Assistant: Read, Review and Verify

### ISSUE THREAD

**Issue by josephjclark** [OpenFn/lightning#2492]
Add a small content warning to the end of AI chats to discourage users from taking AI responses at face value.

Claude has this "Claude can make mistakes" line:

![Image](https://github.com/user-attachments/assets/319c1d6a-40d8-4972-baf6-5175b60694fa)

I want something like this, but I also want to show confidence in our model (well, or prompt actually, but we are abstracting that). Also I'm not so worried about hallucinations or straight up errors - I'm worried about style, about subtle side effects, about whether the user has really engaged with the problem. Because the hardest part of most software is the design, rather than the syntax.

Note that the warning should only appear on the latest chat. So it's sort of pinned to the last reply.

This was penciled in here https://github.com/OpenFn/lightning/pull/2478/files (search for `Read, review and verify`) - you can re-use the template to save time. Although it appears on each message, it doesn't have the "sticky" behaviour.

Suggestions:
* Read, Review, Verify
* This response was generated by an algorithm (still my favourite)
* AI-generated response
* ?

**Comment by josephjclark** [comment]
I'm still not sure about this phrase - "review and verify" basically mean the same thing. I'm still working on it. Keen to get the disclaimer added to the assistant ASAP

**Comment by josephjclark** [comment]
If someone picks up this ticket, can you google around to see if anyone has written any kind of responsible guide to using an LLM, and link it here?

I'm thinking of articles or guidelines or blog posts that provide gentle guidance to users on how to use LLMs. How to ask the right question, how to consider responses, how to be a little bit skeptical, how to not take things at face value.

I don't know if any such thing exists, but it would save me a lot of time if it does :)

**Comment by theroinaochieng** [comment]
Moving this to the icebox.

@christad92  please let me know if you find this critical for the productionizing AI epic. As shown in Joe's comments, this might take a while to resolve and we don't find it a critical blocker in terms of impact to the users at this stage. 
I'll be happy to reassign this based on your feedback.

**Comment by josephjclark** [comment]
Yeah this is fine. This is my feature request, and it's also blocked by me 🤷 I'll work out a solution I'm happy eventually.

But yes, Icebox is appropriate right now.
